tests/core/test_listeners.py

```diff
@@ -64,10 +64,18 @@ def test_listener_module_can_be_specified_via_listeners_list(self):
         })
         self.assertTrue(self.demo_mocked.called)
 
-    def test_callback_called_when_action_is_not_filtered(self):
+    @mock.patch('kinto.core.utils.parse_resource',
+                return_value={'bucket': 'b_id',
+                              'collection': 'c_id'})
+    @mock.patch('kinto.core.utils.view_lookup',
+                return_value=('record',
+                              {'bucket_id': 'b_id', 'collection_id': 'c_id'}))
```

Comment:
Are you sure you need to mock these?

Reply:
The reason why I decided to mock these is, the test (test_callback_called_when_action_is_not_filtered)  was breaking when the newly added class (EventResourceIdFilter) to filter by resource by ID calls view_lookup in utils.py. It expects a upath_info to be passed in request (which can be easily added in test_listener.py) and a non blank uri. I thought that this test is meant to check if the callback is being called or not and not a test view_lookup function. I might be wrong here, as I can surmise from your comment. Please guide me how to handle this blank URI issue then, assuming that adding upath_info in the Request class in test_listener.py file is ok. Also, you made this comment for this test only or for other tests too in this file where I am mocking these 2 functions?
Thanks

Reply:
I might have a bug here, let me take a deeper look.